include/nccl_ofi.h

```diff
@@ -399,16 +399,6 @@ struct nccl_net_ofi_send_comm {
 	int (*regMr)(nccl_net_ofi_send_comm_t *send_comm, void *data, size_t size, int type,
 			      void **mhandle);
 
-	/*
-	 * @brief	Register DMA memory region on send communicator (both Host and CUDA)
-	 *
-	 * This operation is not supported.
-	 *
-	 * @return	Memory handle for data send operations
-	 */
-	int (*regMrDmaBuf)(nccl_net_ofi_send_comm_t *send_comm, void *data, size_t size,
```

Comment:
i don't understand the need to delete this.  Please don't.

Reply:
With the cache interface change, the only difference in logic between the `regMrDmaBuf` path and the `regMr` path is that we pass `FI_MR_DMABUF` on the `fi_mr_regattr` call, so I was hoping to keep the codepath unified.  

Keep in mind that if the plugin advertises dmabuf support, regMrDmabuf may be called by NCCL with `fd=-1` and we are expected to fall back to a regMr implementation. So regardless of whether our internal per-communicator API defines regMrDmaBuf and regMr separately or as a single function, the top-level `plugin->regMrDmaBuf` call is liable to invoke either one and these APIs must cross each other.

If not all the way at the top, where would you put that check?